<launch>
    <!-- Argument -->
    <arg name="config_file" default="$(find msd700_hardware)/config/msd700_odometry.yaml"/>

    <!-- Load param-->
    <rosparam command="load" file="$(arg config_file)" />

    <!-- Odometry node -->
    <node pkg="msd700_hardware" type="odom_node.py" name="odom_node" output="screen" />
</launch>